src/includes/class-alma-wc-plugin.php

```diff
@@ -421,15 +421,27 @@ private function run() {
 		}
 
 		// Don't advertise our payment gateway if we're in test mode and current user is not an admin.
-		if ( $this->settings->get_environment() === 'test' && ! current_user_can( 'administrator' ) ) {
+		if ( ! $this->current_user_can_see_widget() ) {
 			$this->logger->info( 'Not displaying Alma in Test mode to non-admin user' );
-
 			return;
 		}
 
 		$this->init_widget_handlers();
 	}
 
+	/**
+	 * Can the user see the widget ?
+	 *
+	 * @return bool
+	 */
+	public function current_user_can_see_widget() {
```

Comment:
```suggestion
	public function is_allowed_to_see_alma( WP_User $user ) {
```

will be used with `alma_wc_plugin()->is_allowed_to_see_alma( wp_get_current_user() );`

then, we will be able to test it...

So you could also write a test to ensure this methods will works fine now 😘